terraform/environment/dns_caseworker.tf

```diff
@@ -0,0 +1,46 @@
+
+# caseworker front load balancers
+data "aws_route53_zone" "refunds_opg_digital" {
+  provider = aws.old_refunds_production
+  name     = "refunds.opg.digital"
+}
+
+data "aws_elb" "old_production_caseworker_front" {
+  provider = aws.old_refunds_production
+  name     = "caseworker-front-production"
+}
+
+locals {
+  caseworker_url_dns_target_dev_preprod = {
+    alias = {
+      name    = aws_lb.caseworker_front.dns_name
+      zone_id = aws_lb.caseworker_front.zone_id
+    }
+  }
+
+  caseworker_url_dns_target_production = {
+    alias = {
+      name    = data.aws_elb.old_production_caseworker_front.dns_name
+      zone_id = data.aws_elb.old_production_caseworker_front.zone_id
+    }
+  }
+  #this condition is failing to select the correct set of load balancers
+  caseworker_url_dns_target = local.account_name == "production" ? local.caseworker_url_dns_target_production : local.caseworker_url_dns_target_dev_preprod
```

Comment:
is this switching between old and new production formats?

Reply:
it's switching between old and new production targets